### PR TITLE
Remove bower.json and .bowerrc files

### PR DESCRIPTION
**Pull request by abdonrd**

### CODE REVIEW

File: gh/templates/gp.sh [hunk 1]
```diff
@@ -46,6 +46,7 @@ bower install git@$hostname:$org/$repo#$branch
 git checkout ${branch} -- demo
 rm -rf components/$repo/demo
 mv demo components/$repo/
+rm bower.json .bowerrc
```
Comment (robdodson) [inline gh/templates/gp.sh]: could you add this to line 47?

Comment (abdonrd) [inline gh/templates/gp.sh]: Done!

**Comment by samccone** [comment]
Thanks @abdonrd :)

**Comment by abdonrd** [comment]
Your welcome @samccone! :sunny:

**Pull request merged**